src/Basics/loops_hw1.py

```diff
@@ -0,0 +1,119 @@
+from src.Basics.utils import get_random_nums
+
+"""
+	Loops Homework 1
+""" 
+
+def get_maximum(listOfNumbers: list[float]):
+	try:
```

Comment:
I am not sure why we need try catch here. If theres an error, I'd rather it not get caught. Catching an error without correcting it is not the right thing to do.